.github/workflows/mega-linter.yml

```diff
@@ -0,0 +1,81 @@
+  APPLY_FIXES: all # When active, APPLY_FIXES must also be defined as environment variable (in github/workflows/mega-linter.yml or other CI tool)
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: MegaLinter
+    runs-on: ubuntu-latest
+    steps:
+      # Git Checkout
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      # MegaLinter
+      - name: MegaLinter
+        id: ml
+        # You can override MegaLinter flavor used to have faster performances
+        # More info at https://megalinter.github.io/flavors/
+        uses: megalinter/megalinter@v5
+        env:
+          # All available variables are described in documentation
+          # https://megalinter.github.io/configuration/
+          VALIDATE_ALL_CODEBASE: false # ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }} # Validates all source when push on master, else just the git diff with master. Override with true if you always want to lint all sources
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # ADD YOUR CUSTOM ENV VARIABLES HERE OR DEFINE THEM IN A FILE .mega-linter.yml AT THE ROOT OF YOUR REPOSITORY
+          # DISABLE: COPYPASTE,SPELL # Uncomment to disable copy-paste and spell checks
+
+      # Create or delete the cleanup branch
+      - name: Update cleanup branch
+        if: ${{ github.event.repository.owner.login == 'AliceO2Group' }}
+        env:
+          REMOTE_URL: "https://alibuild:${{ secrets.alibuild_github_token }}@\
+            github.com/alibuild/${{ github.event.repository.name }}"
+          # An empty CLEANUP_COMMIT means delete the branch.
+          CLEANUP_COMMIT: ""
+        run: |
+          if [ "${{ steps.ml.outputs.has_updated_sources }}" = 1 ]
+          then
+            CLEANUP_COMMIT="HEAD"
+          fi
+          git push -f "$REMOTE_URL" "$CLEANUP_COMMIT:refs/heads/alibot-cleanup-ml-$PR_NUMBER"
+
+      - name: Create pull request with applied fixes
+        uses: alisw/pull-request@v2
+        if: steps.ml.outputs.has_updated_sources == 1 && ${{ github.event.repository.owner.login == 'AliceO2Group' }}
+        with:
+          source_branch: 'alibuild:alibot-cleanup-ml-${{ env.PR_NUMBER }}'
+          destination_branch: '${{ github.event.pull_request.head.label }}'
+          github_token: ${{ secrets.alibuild_github_token }}
+          pr_title: >-
+            [MegaLinter] Apply linters automatic fixes to
+            #${{ env.PR_NUMBER }}
+          pr_body: >-
+            This PR cannot be merged as is. You should either run MegaLinter
+            yourself and update the pull request, or merge this PR in yours.
+
+            You can find how to run MegaLinter locally at
+            <https://megalinter.github.io/latest/mega-linter-runner/>.
```

Comment:
This phrasing isn't ideal in the other workflows either, but I find this a little confusingly worded. Maybe:

```
Your PR ${{ github.event.repository.full_name }}#${{ env.PR_NUMBER }}
cannot be merged as is.
...
```